### DISCUSSION TITLE
Помогите настроить экспортер на определенный кластер сервера 1С

### DISCUSSION THREAD

**Discussion by AlekseyStorozhuk** [LazarenkoA/prometheus_1C_exporter#45] [Q&A]
доброго времени суток!
Не могу победить ситуацию...
На одном сервере 1С есть несколько кластеров
И экспортер видит только последний...
Как мне настроить на конкретный кластер или увидеть все? Куда прописать UID кластера?...

**Comment by LazarenkoA** [comment]
UID кластера не нужно прописыввать никуда. Покажи текущий конфиг.

**Comment by AlekseyStorozhuk** [comment]
[settings.zip](https://github.com/user-attachments/files/17401202/settings.zip)

**Comment by AlekseyStorozhuk** [comment]
![Снимок](https://github.com/user-attachments/assets/f6cafad3-b2db-4f91-b414-2e6b2ed586b7)
такая настройка кластеров
Сервер на порту 1540

Я вижу метрики только кластера Test на 1241 порту... Если кластер удаляю, метрики собираю с 1541

**Comment by LazarenkoA** [comment]
да, экспортер получает последний кластер, сейчас нет возможности обрабатывать разные кластера

**Comment by AlekseyStorozhuk** [comment]
а не планируется через параметр добавить? Было бы удобно...

**Comment by LazarenkoA** [comment]
не планировал. Сделать можно

**Comment by AlekseyStorozhuk** [comment]
Все взлетело... Я принудительно идентификатор прописал... По красоте конечно надо сделать через параметр... Но.... Я не сильно шарю в Go... Боюсь закопаюсь...
Модуль для правки BaseExplorer.go. Передача идентификатора кластера - в строке 279

А за работу Вашу, спасибо огромное. Активно пользую эту разработку, 1С2GIT и LogViewer...

**Comment by zakharovy** [comment]
Уважаемый Артем. Экспортер - чудо, у нас все отлично работало, пока мы не придумали делать несколько кластеров :))
Вы уже отвечали, но всё-же не появилось - ли у вас намерение таки добавить поддержку нескольких кластеров на одном стервере?
Это было-бы просто восхитительно.

**Reply by LazarenkoA** [reply]
Нет, забыл про эту проблему, постораюсь сделать на следующей неделе

**Comment by DS-Krylov** [comment]
Добрый день, планируем внедрять ваш экспортер как типовое решение в нашей организации, но у нас 7 кластеров
Подскажите пожалуйста, экспортер в итоге получил поддержку нескольких кластеров по умолчанию?
На предварительных тестах выглядит как будто нет, но возможно надо как то специфично настроить ещё
